packages/graph/hash_graph/lib/graph/src/api/rest/data_type.rs

```diff
@@ -96,7 +96,7 @@ impl RoutedResource for DataTypeResource {
     }
 }
 
-#[derive(Serialize, Deserialize, ToSchema)]
+#[derive(Debug, Serialize, Deserialize, ToSchema)]
```

Comment:
I've tried to only have the requests pass their (potentially long) arguments. This means that we need debug on the Rest body structs for them to be put into the instrumented traces